responsibleai/tests/rai_insights/test_rai_insights_save_and_load_scenarios.py

```diff
@@ -207,11 +207,8 @@ def test_rai_insights_add_save_load_save(self, manager_type, adult_data):
             )
         elif manager_type == ManagerNames.ERROR_ANALYSIS:
             rai_insights.error_analysis.add()
-        elif manager_type == ManagerNames.EXPLAINER:
```

Comment:
Nit: Why is this change needed? Won't the value error double check that an undesired enum doesn't go through, and won't we be assuming with the `else` that we have to add in the explainer? Perhaps not a big deal coz this is a testing file.

Reply:
Would also like to know :-)

Reply:
The new manager would be added anyways to the parameter list and the code to handle it would be added in the same PR, isn't it? It is dead code anyways.